_episodes/11-tools.md

```diff
@@ -0,0 +1,46 @@
+--- 
+title: Graphical User Interface (GUIs) 
+teaching: 5 
+exercises: 0
+questions: 
+- "Are there Git GUI applications that I can use?" 
+objectives: 
+- "See an example of a Git GUI application" 
+- "Get a list of popular Git GUI applications." 
+keypoints: 
+- "There are a number of GUI applications available for working with Git." ---
+
+There are a number of different graphical user interface (GUI) apps for Git
+available which many people prefer for their daily work. You might wonder why we
+had you start with the command line, if GUIs are available? The main reason is
+that most GUIs only contain a subset of Git functionality and knowing how to use
+the command line is necessary to use and understand Git at its' fullest
+capacity.
+
+## GUI Application Example
+## 
+Here is an example of a popular GUI called GitKraken being used to perform the
+following: 
+*   creating a new branch
+*   staging a file
+*   performing a commit
+*   checking out a branch
+*   merging a branch
+*   deleting a branch
+
+![GitKraken demo](../fig/gitkraken.gif)
+
+## Popular Free Applications
+
+| Application        | Platforms          | Link                    |
+|:-------------------|:-------------------|:------------------------|
+| GitKraken | Windows, Mac, Linux | https://www.gitkraken.com |
+| SourceTree | Windows, Mac | https://www.sourcetreeapp.com |
+| GitHub Desktop | Windows, Mac | https://desktop.github.com |
+| Magit | Windows, Mac, Linux | https://magit.vc |
```

Comment:
omg <3

Reply:
i want to reject it because Vim is being discriminated against, but I'll allow it just this once